### DISCUSSION TITLE
File Changed Tab shows "Oops, something went wrong. Retry" Error

### DISCUSSION THREAD

**Discussion by DanJ210** [community/community#37991] [Pull Requests]
I've been getting this for the past couple of weeks now on multiple machines, same with my other developer teammates. I can't review most PR's fully because they won't fully load and click "Retry" never resolves the issue. The PR's don't have to be that large either, just a certain size is always loaded at first but the rest will never load after scrolling past that point.

![image](https://user-images.githubusercontent.com/16159685/199274052-723f15de-189f-4d1c-8815-3a8a48974418.png)

**Comment by ossanna16** [comment]
Hi there @DanJ210 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Reply by DanJ210** [reply]
Are they any investigations going on over this? I'm just using Edge on web and still having this issue.

**Reply by SuchyTheMaster** [reply]
Me too (on all browsers). It blocks code review and have material impact in my company.

**Comment by winston-spruce** [comment]
I am having exactly the same issue.  I wrote to customer support and I haven't heard anything.   It is preventing me from being able to view code online.   This issue is having a material impact to my work.

**Comment by alsyundawy** [comment]
I am having exactly the same issue

**Comment by marcindulak** [comment]
Unfortunately, I'm also having this problem since yesterday, from different machines, operating systems and networks, in both firefox and google chrome. It's a private repo, a pull request with several thousands lines and a few hundreds comments.

The developer tools network tab show a HTTP 502 error
```
Request URL:
https://github.com/user/repo/diffs/1?base_sha=xxx&commentable=true&name=branch&pull_number=1&sha1=xxx&sha2=xxx&w=false
Request Method: GET
Status Code: 502 Bad Gateway
Remote Address: 140.82.121.4:443
Referrer Policy: strict-origin-when-cross-origin
```

The console shows as the initiator
```
window.IncludeFragmentElement.fetch | @ | include-fragment-element-hacks.ts:5
```

Update: after pushing more commits to the PR, the file previously giving `Oops, something went wrong. Retry` is loading again.

Another unresolved discussion about this problem https://github.com/orgs/community/discussions/78920

**Comment by pjcpjc** [comment]
I'm experiencing this issue. It's having a material impact on my work. 

The code filter (which is quite primitive) doesn't fix this. For example, the number of code changes for my .py files is small, but I get the same problem applying the .py file filter.

Is there anything we can do to get the GitHub team to care about this issue? Alternately, are other tools like BitBucket willing to provide a professional experience?

**Comment by Shautora** [comment]
Just adding that I'm experiencing the same issue, and it's blocking me from completing my code review.  Until yesterday, sometimes clicking reload 10-20 times would work, but now, even that doesn't seem to help.  The inspector gives the following error:
```
Content-Security-Policy: The page’s settings blocked a JavaScript eval (script-src) from being executed because it violates the following directive: “script-src https://github.githubassets.com/” (Missing 'unsafe-eval')
Uncaught (in promise) Error: Failed to load resource: the server responded with a status of 404
    h include-fragment-element.js:199
```

- **Status:**  404
- **Version:**  HTTP/2
- **Transferred:**  5.01 kB (0 B size)
- **Referrer Policy:**  strict-origin-when-cross-origin
- **DNS Resolution:**  System
